Source/ProgressionSystemRuntime/Private/Widgets/ProgressionMenuWidget.cpp

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#include "Widgets/ProgressionMenuWidget.h"
+//---
+
+//---
+#include <string>
```

Comment:
Don't use this std string include. UE already includes own FString in all base classes by default instead of std one. Remove it from all your files

Reply:
done

Reply:
<string> is still there for Save Widget

Reply:
<string> is still there for Save Widget

Reply:
`<string>` is still included in the `ProgressionMenuWidget.cpp`